### ISSUE TITLE
PFE-Dropdown has incorrect aria and a11y setup

### ISSUE THREAD

**Issue by wesruv** [patternfly/patternfly-elements#2054]
## Description of the issue
This was noticed on PFE 1.x, but I suspect it might also be in 2.x, full discussion is here: https://chat.google.com/room/AAAA0siaVdA/V0r-JRHGmOY

It should follow the aria/accessibility setup from: https://pattern-library.dequelabs.com/components/selects

Basically it should be following the `listbox` pattern, instead of a `menu`, `menu` has very specific expectations similar to the "File" menu in applications:
> While the term "menu" is generically used a term used to describe site navigation, the menu role is for a list of action or functions that require complex functionality, such as composite widget focus management and first-character navigation
From: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/menu_role

### Impacted component(s)

- pfe-dropdown

**Comment by wesruv** [comment]
I don't see this component in RHDS yet, adding the tag because I'm guessing it might be headed there?

**Comment by bennypowers** [comment]
The next step for pfe-dropdown will be to align it with PFv4. We could address this during that process. Is it the case that you will want dropdown in RHDS, being the successor to pfe 1.x dropdown? in that case please [fill out a request for new component](https://github.com/RedHat-UX/red-hat-design-system/issues/new?assignees=methomps&labels=discovery+needed%2Cdesign+needed%2Cnew+component&template=1-new-component.yml&title=[feat]+dropdown component) there.

**Comment by wesruv** [comment]
Cool, I think it'd be great to have this in RHDS, I'd guess @kylebuch8 && @TylerMartin5 would also be interested. Made a ticket: https://github.com/RedHat-UX/red-hat-design-system/issues/425

**Comment by bennypowers** [comment]
See also https://github.com/patternfly/patternfly-elements/issues/2049

**Comment by bennypowers** [comment]
Closing, as we have `<pf-select>` now.
